flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/handlers/JarPlanHandler.java

```diff
@@ -65,7 +65,7 @@ public JarPlanHandler(Executor executor, File jarDirectory) {
 					return writer.toString();
 				}
 				catch (Exception e) {
-					throw new FlinkFutureException(e);
+					throw new CompletionException(e);
```

Comment:
Why it's not wrapping a `FlinkException` here?

Reply:
Because we didn't add any additional information to `e`.